### ISSUE TITLE
`gsutil stat` error message does not match other command error messages

### ISSUE THREAD

**Issue by vapier** [GoogleCloudPlatform/gsutil#288]
most gsutil commands have an error message that looks like:
```
$ gsutil cp /alsjdf gs://chromiumos-sdk/alskdjf
CommandException: No URLs matched: /alsjdf
```

but the stat command looks completely different.  in 4.13, it looks like:
```
$ gsutil stat gs://chromiumos-sdk/alskdjf
INFO 0714 22:40:43.256200 stat.py] No URLs matched gs://chromiumos-sdk/alskdjf
```

would it be possible to revise that output to look like the others ?  i know the other commands are throwing exceptions internally, but still, we should be able to do something here to improve it ?

**Comment by thobrla** [comment]
Thanks for reporting this - this change in output format is a bug, likely caused by https://github.com/GoogleCloudPlatform/gsutil/commit/3b9dac53b220b62e71254b8f53987af5b4f3f4eb .

**Comment by vapier** [comment]
that did fix b/16020252 as requested (use stderr instead of stdout for error messages), but it also changed the format of the message

**Comment by thobrla** [comment]
Yes, the format change was an unintended side effect that went unnoticed because the tests just check only that "No URLs matched %s" is contained in the output.
